include/bluetooth/audio/mcs.h

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (c) 2019 - 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
I am not sure yet what is best.
I can certainly move this to be internal for now - I expect it is easier to to add to the public interface later than to remove from it.
But for consistency, should the same also apply for the client header (mcc.h)?

The file is marked as experimental, so I believe it can also be left as is for now, and moved when we have decided on the API.
What do you prefer?

Reply:
As long as media_proxy.h relies on this, this should remain in the public structure, but I don't think it is the intention that that media_proxy.h relies on this, right? 

Regarding if this should apply to mcc.h, that depends on whether we only want to expose the MCS and MCC functionality via the Media Proxy, or if we want to allow users to use those implementations directly.